plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/app/PluginStrings.js

```diff
@@ -232,6 +232,9 @@ Ext.define('NX.coreui.app.PluginStrings', {
     ADMIN_BLOBSTORES_LIST_NEW_BUTTON: 'Create blobstore',
     ADMIN_BLOBSTORES_LIST_NAME_COLUMN: 'Name',
     ADMIN_BLOBSTORES_LIST_TYPE_COLUMN: 'Type',
+    ADMIN_BLOBSTORES_LIST_BLOB_COUNT_COLUMN: 'Blob Count',
```

Comment:
A blob can be almost anything, right? Which is why we can’t just say “Component Count” here?

Probably should have a brief explanation of what a blob/blobstore is, in the UI.